pkg/reconciler/githubsource/webhook_client.go

```diff
@@ -37,29 +37,24 @@ type webhookOptions struct {
 }
 
 type webhookClient interface {
-	Create(ctx context.Context, options *webhookOptions) (string, error)
-	Delete(ctx context.Context, options *webhookOptions, hookID string) error
-	CreateWithGitHubBaseURL(ctx context.Context, options *webhookOptions, alternateGitHubAPIURL string) (string, error)
-	DeleteWithGitHubBaseURL(ctx context.Context, options *webhookOptions, hookID string, alternateGitHubAPIURL string) error
+	Create(ctx context.Context, options *webhookOptions, alternateGitHubAPIURL string) (string, error)
+	Delete(ctx context.Context, options *webhookOptions, hookID, alternateGitHubAPIURL string) error
 }
 
 type gitHubWebhookClient struct{}
 
-func (client gitHubWebhookClient) Create(ctx context.Context, options *webhookOptions) (string, error) {
-	return client.CreateWithGitHubBaseURL(ctx, options, "")
-}
-
-func (client gitHubWebhookClient) Delete(ctx context.Context, options *webhookOptions, hookID string) error {
-	return client.DeleteWithGitHubBaseURL(ctx, options, hookID, "")
-}
-
-func (client gitHubWebhookClient) CreateWithGitHubBaseURL(ctx context.Context, options *webhookOptions, alternateGitHubAPIURL string) (string, error) {
+func (client gitHubWebhookClient) Create(ctx context.Context, options *webhookOptions, alternateGitHubAPIURL string) (string, error) {
 	logger := logging.FromContext(ctx)
 
 	ghClient := client.createGitHubClient(ctx, options)
 	if alternateGitHubAPIURL != "" {
 		//This is to support GitHub Enterprise, this might be something like https://github.company.com/api/v3/
-		ghClient.BaseURL, _ = url.Parse(alternateGitHubAPIURL)
+		baseURL, err := url.Parse(alternateGitHubAPIURL)
+		if err != nil {
+			logger.Infof("Failed to create webhook - Error occured parsing githubAPIURL %s, error was %v", alternateGitHubAPIURL, err)
+			return "", fmt.Errorf("Error occured parsing githubAPIURL: %v", err)
```

Comment:
Lowercase 'Error'. 

See https://github.com/golang/go/wiki/Errors: "Error strings should not start with a capital letter because they'll often be prefixed before printing"

Reply:
corrected